pkgs/development/python-modules/python-irodsclient/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchPypi, prettytable, six }:
+
+buildPythonPackage rec {
+  pname = "python-irodsclient";
+  version = "0.8.2";
+  
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "e77c345bea028c748047656f35f576210528b1332087b5f0a7e2ddd179301447";
+  };
+  
+  # xmlrunner is not necessary
+  patchPhase = ''
+    substituteInPlace setup.py --replace "'xmlrunner>=1.7.7'" ""
+  '';
+  
+  propagatedBuildInputs = [ prettytable six ];
+  
+  # Test depends on existing iRODS installation
+  doCheck = false;
```

"in lieu" of tests, please do
```suggestion
  doCheck = false;
  pythonImportsCheck = [ "irods" ];
```